test/clj/cider/nrepl/middleware/util/instrument_test.clj

```diff
@@ -41,7 +22,7 @@
     '#{[(new java.lang.Integer 1) []]}))
 
 (deftest instrument-clauses-test
-  (are [exp res] (clojure.set/subset? res (breakpoint-tester exp))
+  (are [exp res] (clojure.set/subset? res (t/breakpoint-tester exp))
```

Comment:
Should `(:require [clojure.set :as set])` in the `ns` form

Reply:
I'm pushing a fix for this too.